### DISCUSSION TITLE
deployment planning for multiable sites to central SOC

### DISCUSSION THREAD

**Discussion by ghostwalker42** [Security-Onion-Solutions/securityonion#9816] [Unsupported Versions]
Hi All,

 I don't know if this the right place for this. I work at an MSP and we are looking to build out a SOC. I have been doing some digging and I think Security Onion would be great for us since it has so many tools. 

My question is We have around 10 Sites we support and in each site we have around 10-15 servers. I know I can do a collector node in each site and forward the data back to a central Manager/DB. The real question is How much storage should I be planning for if I don't want to do pcap files from each site. 

I was thinking I would want to forward all Event data from out servers and about 10-20 High risk users. I also wanted to put out 1 or 2 honeypots in each site. 

Should I be planning for 3-4 TB per site or more, Please let me know your thoughts. 

Thanks in advance.

**Comment by Jaap79** [comment]
The way we solved a similar usecase:

- Build a Distributed deployment, one Manager (obviously) and a slew of Search Nodes in your central location / HQ / main office.
- Deploy one Sensor node per site
- We're using ZeroTier to connect the whole thing together. Mix in a firewall that runs Zerotier natively (like OPNsense if you have that option, else you might want to add the whole bunch to the same ZeroTier network directly)

I can´t say much about site sizing. It depends on what you do and do not wish to capture. The PCAPs remain on the Sensor nodes anyway, the load to the Manager node in our case is about 1 to 3 Mbps by average every few minutes. Nothing too special.

If it's around 20 servers / bunch of users and network equipment, a 1TiB drive would go a long way if you drop all the PCAP related grabs.

**Reply by ghostwalker42** [reply]
Thank you for your feedback on this. This was very helpful to see how some others are deploying it in similar configurations.
